code/frontend/src/styles/_accents.scss

```
@@ -0,0 +1,94 @@
+// =============================================================================
+// Accent Presets
+// Each preset overrides the --brand-* scale and --accent-rgb triple.
+// The 'default' preset keeps the UI brand palette (from _tokens.scss) unchanged
+// and restores the logo's original hexes. Scales follow the same perceptual ramp
+// (Tailwind-style 50..950 lightness).
+//
```

Stylelint (17.9.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)